configs/keycloak_bridge.yml

```diff
@@ -7,7 +7,6 @@ management-http-host-port: 0.0.0.0:8877
 account-http-host-port: 0.0.0.0:8866
 register-http-host-port: 0.0.0.0:8855
 mobile-http-host-port: 0.0.0.0:8844
-monitoring-http-host-port: 0.0.0.0:8899
```

Comment:
Why is this line removed?

Reply:
The content of the files comes from the dev-env --> the monitoring is not listed in this config file.
I'll check with @harture and fix where appropriate.